action.yaml

```diff
@@ -0,0 +1,62 @@
+name: 'fluvio'
+description: 'Installs Fluvio CLI from source with required dependencies (e.g. Helm, Rust) and starts a Local Cluster. Used for developing fluvio.'
+inputs:
+  spus:  # number of SPUs
+    description: 'number of SPUs to create for local cluster'
+    required: false
+    default: "1"
+  cluster-type:
+    description: 'Type of cluster (i.e. local or kubernetes)'
+    required: true
+    default: 'local'
+  log-options:
+    description: "Add rust log options (e.g. debug, warn, info)"
+    required: false
+    default: 'info'
+  version:
+    description: "Use a specific version of the fluvio CLI (e.g. latest, 0.6.0)"
+    required: false
+    default: 'latest'
+  development:
+    description: "Use a debug release version of fluvio"
+    required: false
+    default: "true"
+  platform:
```

Comment:
I don't think there is a good way to map the OS name "ubuntu-latest", "macOS-latest", or "Linux" to platform name in the workflow. Rather than accepting "platform" as an input, is is best computed in the shell scripts used by this action. You would just need to add to `action-install-minikube.sh ` the same mapping logic in `ci-replace-helm.sh`.

Reply:
Commit 168c946 removes; I've added the case to the minikube install; but I was also thinking that maybe we should replace the case with the following for determining OS, using `$OSTYPE`:


```bash

if [[ "$OSTYPE" == "linux-gnu"* ]]; then
       # Install for Linux
elif [[ "$OSTYPE" == "darwin"* ]]; then
        # Mac OSX
elif [[ "$OSTYPE" == "cygwin" ]]; then
        # POSIX compatibility layer and Linux environment emulation for Windows
elif [[ "$OSTYPE" == "msys" ]]; then
        # Lightweight shell and GNU utilities compiled for Windows (part of MinGW)
elif [[ "$OSTYPE" == "win32" ]]; then
        # I'm not sure this can happen.
elif [[ "$OSTYPE" == "freebsd"* ]]; then
        # ...
else
        # Unknown.
fi
```

Reply:
That would be a slight improvement if `$OSTYPE` is reliable and you should also check the CPU arch. But I think this is good for now.